### ISSUE TITLE
[Bug] unable to convert model from mobilenet-v2-pytorch

### ISSUE THREAD

**Issue by guoyejun** [openvinotoolkit/openvino#4068]
- OpenVINO => 2020.3.341 installed with l_openvino_toolkit_p_2020.3.341.tgz
- Operating System / Platform => Ubuntu 18.04.5 LTS (GNU/Linux 4.15.0-128-generic x86_64)
- Compiler => gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)
- Problem classification: Model Conversion
- Framework: Torch
- Model name: MobileNet

##### Detailed description
I'm trying the steps at https://docs.openvinotoolkit.org/latest/pot_configs_examples_README.html, and there's issue at the model convert part.

##### Steps to reproduce

Just follow https://docs.openvinotoolkit.org/latest/pot_configs_examples_README.html, installed openvino and the following checks are done with no errors.
```
mo.py -h
accuracy_check -h
pot -h
```

downloaded mobilenet-v2-pytorch as the document said:
```
# sha256sum mobilenet-v2.pth
ecbe2b568c8602549fa9e1d5833c63848f490a48d92e5d224d1eb2063e152cf8  mobilenet-v2.pth
# sha256sum MobileNetV2.py
7071e01e07e5f1b6127a2628d1be9c65cc20edfa745f4f4eb6354e7b27f5b444  MobileNetV2.py
```

then, tried the model convert:
```
~/openvino_work/example# python /opt/intel/openvino/deployment_tools/open_model_zoo/tools/downloader/converter.py --name mobilenet-v2-pytorch --precisions FP32
========= Converting mobilenet-v2-pytorch to ONNX
Conversion to ONNX command: /root/pyenv/ov/bin/python /opt/intel/openvino/deployment_tools/open_model_zoo/tools/downloader/pytorch_to_onnx.py --model-name=MobileNetV2 --model-path=/root/openvino_work/example/public/mobilenet-v2-pytorch --weights=/root/openvino_work/example/public/mobilenet-v2-pytorch/mobilenet-v2.pth --import-module=MobileNetV2 --input-shape=1,3,224,224 --output-file=/root/openvino_work/example/public/mobilenet-v2-pytorch/mobilenet-v2.onnx --input-names=data --output-names=prob

ONNX check passed successfully.

========= Converting mobilenet-v2-pytorch to IR (FP32)
Conversion command: /root/pyenv/ov/bin/python -- /opt/intel/openvino_2020.3.341/deployment_tools/model_optimizer/mo.py --framework=onnx --data_type=FP32 --output_dir=/root/openvino_work/example/public/mobilenet-v2-pytorch/FP32 --model_name=mobilenet-v2-pytorch --input=data '--mean_values=data[123.675,116.28,103.53]' '--scale_values=data[58.624,57.12,57.375]' --reverse_input_channels --output=prob --input_model=/root/openvino_work/example/public/mobilenet-v2-pytorch/mobilenet-v2.onnx

Model Optimizer arguments:
Common parameters:
        - Path to the Input Model:      /root/openvino_work/example/public/mobilenet-v2-pytorch/mobilenet-v2.onnx
        - Path for generated IR:        /root/openvino_work/example/public/mobilenet-v2-pytorch/FP32
        - IR output name:       mobilenet-v2-pytorch
        - Log level:    ERROR
        - Batch:        Not specified, inherited from the model
        - Input layers:         data
        - Output layers:        prob
        - Input shapes:         Not specified, inherited from the model
        - Mean values:  data[123.675,116.28,103.53]
        - Scale values:         data[58.624,57.12,57.375]
        - Scale factor:         Not specified
        - Precision of IR:      FP32
        - Enable fusing:        True
        - Enable grouped convolutions fusing:   True
        - Move mean values to preprocess section:       False
        - Reverse input channels:       True
ONNX specific parameters:
Model Optimizer version:
[ ERROR ]  Exception occurred during running replacer "REPLACEMENT_ID" (<class 'extensions.front.user_data_repack.UserDataRepack'>): No node with name prob.
 For more information please refer to Model Optimizer FAQ (https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_Model_Optimizer_FAQ.html), question #51.
FAILED:
mobilenet-v2-pytorch
```

If I do not set '--precisions FP32', FP16 will be firstly tried and the same issue reported: No node with name prob.

I also uploaded the converted mobilenet-v2.onnx here.
[mobilenet-v2.zip](https://github.com/openvinotoolkit/openvino/files/5887370/mobilenet-v2.zip)

**Comment by guoyejun** [comment]
it would be nice if your side works well and provide the converted FP32 model as a quick reply, so I can continue the next steps to try with pot, thanks.

**Comment by guoyejun** [comment]
btw, i tried the next step for the performance benchmarking with onnx format according to the webpage, and there's also an issue.

```
# python /opt/intel/openvino/deployment_tools/tools/benchmark_tool/benchmark_app.py -m /root/openvino_work/example/public/mobilenet-v2-pytorch/mobilenet-v2.onnx
[Step 1/11] Parsing and validating input arguments
Traceback (most recent call last):
  File "/opt/intel/openvino/deployment_tools/tools/benchmark_tool/benchmark_app.py", line 22, in <module>
    sys.exit(main() or 0)
  File "/opt/intel/openvino_2020.3.341/python/python3.7/openvino/tools/benchmark/main.py", line 20, in main
    run(parse_args())
  File "/opt/intel/openvino_2020.3.341/python/python3.7/openvino/tools/benchmark/parameters.py", line 103, in parse_args
    validate_args(parsed_args)
  File "/opt/intel/openvino_2020.3.341/python/python3.7/openvino/tools/benchmark/parameters.py", line 23, in validate_args
    raise Exception('Path {} is not xml file.')
Exception: Path {} is not xml file.

```

**Comment by jgespino** [comment]
Hi @guoyejun 

This seems to be a bug with the OpenVINO 2020.3.1 LTS version and it does not reproduce on the latest OpenVINO 2021.2 release. Let me take a closer look and get back to you.

Regarding the benchmark_app running ONNX, this will not work on OpenVINO 2020.3.1 LTS as support for reading ONNX model without needing to convert to Intermediate representation was just added on OpenVINO 2021.2 release. 
 
Regards,
Jesus

**Comment by guoyejun** [comment]
got it, thanks.

**Comment by jgespino** [comment]
@guoyejun

This was fixed in 2021.1 as there were changes to the MO preference output node names. If you want to stay on 2020.3 release try applying the workaround that was revered in this request: https://github.com/openvinotoolkit/open_model_zoo/pull/1689

Regards,
Jesus
